configure.ac

```diff
@@ -561,6 +561,37 @@ case "$enable_unwind_ptrace" in
       ;;
 esac
 
+AC_ARG_ENABLE([iberty],
```

Comment:
No. `libiberty` isn't just about demangling. [`libiberty`](https://en.wikipedia.org/wiki/Libiberty) is a generic support library for the GNU toolchain in which demangling is one of the main features. I said to name the keyword after the feature, that is, "`--enable-demangle`". Maybe we can allow specifications like `--enable-demangle=libiberty` in the future if we support other library/API for the demangling purpose.

Reply:
@MrCirdo Just to give you an idea: When an optional feature introduces a library dependency, we name the "`--enable-`" option after the feature, not after the library. For example the `--enable-unicode` option in htop and not `--enable-libncursesw` (it's not `--enable-ncursesw` either). The "`--with-`" option naming is to specify dependencies, thus `--with-libncursesw` would make sense.

* For software that *provides* the library rather than *depend on* them, `--enable-libXXX` would be used. Example: In GCC's build system there are `--enable-libstdcxx`, `--enable-libatomic`, `--enable-libquadmath` etc. because those libraries are *provided* by GCC.

Reply:
Thank you very much for the explanation @Explorer09. I will try one more time 😄